### ISSUE TITLE
randomly dataSourceEof

### ISSUE THREAD

**Issue by skajake** [tumtumtum/StreamingKit#322]
Ever since iOS 10 I have noticed an uptick in dataSourceEof errors while live streaming.

This causes the AutoRecoveringHTTPDataSource to shut down and the stream simply stops playing.

**Comment by codeZhatty** [comment]
I have a same problem. Do you find solution?

**Comment by skajake** [comment]
No, sadly all I can think to do is restart the stream, although this is a less than ideal solution for the end user.

**Comment by leo150** [comment]
Same error trying to play song from server

> dataSourceEof
> attemptReconnect 6087392/12252390

And stream starts from beginning

**Comment by diegostamigni** [comment]
Hi @skajake 
Can you please provide some example or steps to reproduce this issue ?
